Shared/Core/BoneController.cs

```diff
@@ -551,6 +554,12 @@ private void ApplyEffects()
                     FixBustGravity();
                 }
             }
+
+            if (_effectsToUpdate.Count > 2 * numEffects)
```

Comment:
I don't think this is the case. Having another plugin that affects the same bones would not increase `_effectsToUpdate.Count`. It would increase the sizes of inner lists instead. Also it would increase `numEffects`, so it would cause `_effectsToUpdate` to be cleared less often, if anything.

Now admittedly this is all confusing...  The idea was that I wanted to prevent `_effectsToUpdate` from growing indefinitely.